src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -858,17 +858,12 @@ public function convert()
      */
     public static function getAllPodcasts($num_results = 0, $page = 1, $includeSuspended = true)
     {
-        $query = "SELECT podcast_id FROM uryplayer.podcast
-                  ORDER BY submitted DESC ";
-
-        if (!includeSuspended) {
-            $query .= "WHERE suspended = false ";
-        }
-
+        $where .= !$includeSuspended ? 'WHERE suspended = false ': '';
```

Comment:
Surprised `.=` on a brand new variable works at all..

Reply:
Removed it anyway, just to be safe.